src/Libraries/RevitNodes/Elements/DirectShape.cs

```diff
@@ -40,13 +40,6 @@ public override void GetObjectData(SerializationInfo info, StreamingContext cont
             this.syncId = syncId;
             this.materialId = materialId.Value;
         }
-
-        public DirectShapeState(SerializationInfo info, StreamingContext context) :
```

Comment:
Won't this cause a breakage without obsolete message? Not that we need to keep this, but perhaps document somewhere for package developers?